lib/puppet/modulebuilder/builder.rb

```diff
@@ -251,20 +250,11 @@ def build_package
     def ignored_files
       require 'pathspec'
 
-      @ignored_files ||=
-        begin
-          ignored = if ignore_file.nil?
-                      PathSpec.new
-                    else
-                      PathSpec.new(read_file(ignore_file, open_args: 'rb:UTF-8'))
-                    end
-
-          ignored = ignored.add("/#{File.basename(destination)}/") if File.realdirpath(destination).start_with?(File.realdirpath(source))
+      ignored = PathSpec.new
```

Comment:
I don't think the order matters, so you can add `IGNORED` in the constructor:
```suggestion
      ignored = PathSpec.new(IGNORED)
```